### PR TITLE
Added array sum and array product.

### PR DESCRIPTION
**Pull request by fps8k**
## Environment

- Compiler version: 0.8.7
- Target EVM version (as per compiler settings): default
- Framework/IDE (e.g. Truffle or Remix): Remix

- Operating system: Windows

## I added a contract that shows how array addition and multiplication are done.

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/Solidity/pull/10"></a>

### CODE REVIEW

File: Maths/Array functions/AddArray.sol [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            // On every element, add the value of the element to the current sum.
+            // For sums > the maximum uint value, solidity versions > 0.8.0 will revert on overflow.
+            sum += arr[i];
+        }
+
+        // Return the total.
+        total = sum;
+    }
+}
```
Comment (Panquesito7) [inline Maths/Array functions/AddArray.sol:37]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: Maths/Array functions/MulArray.sol [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        {
+            // On every element, multiply the current _product with the value of the elementa.
+            // For products > the maximum uint value, solidity versions > 0.8.0 will revert on overflow.
+            _product *= arr[i];
+        }
+
+        // Return the product.
+        product = _product;
+    }
+}
```
Comment (Panquesito7) [inline Maths/Array functions/MulArray.sol:38]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by mkubdev — APPROVED** [review]
Hey, look good to me 👍🏻

**Comment by fps8k** [comment]
> Hey, look good to me 👍🏻

I am glad it does look good to you 😊.

**Pull request merged**